charts/cluster-scanner/values.yaml

```diff
@@ -218,6 +218,25 @@ imageSbomExtractor:
     #   sentinelAddress: ""
     #   ttl: "168h" # Default ttl is 7 days
 
+  # Provide optional registry mirrors configuration to be used by Image SBOM Extractor to pull images.
+  # [Only Docker HUB images](https://docs.docker.com/registry/recipes/mirror/#gotcha) are going to be pulled from
+  # the provided mirrors.
```

Comment:
The wording here is a bit confusing for me, but perhaps it's just a lack of expertise in this area.
> Provide optional registry mirrors configuration ... to pull images. Only Docker HUB images are going to be pulled ...

☝🏻 Does this mean any provided registries that are not docker hub will be ignored?

Reply:
Thanks @aroberts87 for your comment!

>☝🏻 Does this mean any provided registries that are not docker hub will be ignored?

Not exactly, let me make an example:

Assume you want to pull `alpine:latest` and you have configured `mirror.mycompany.com` as a mirror. Given that this is an image served by Docker Hub, we will contact the mirror you configured and get the image from there. The mirror will only pull the image from Docker Hub if it does not have a local copy yet.

On the other hand, assume you want to pull `quay.io/sysdig/admission-controller:master`. This comes from `quay.io` not Docker Hub. In this case, we will not contact the mirror at all, but will directly pull the image from `quay.io`.

This behavior is consistent with what Docker does (see https://docs.docker.com/registry/recipes/mirror/#gotcha):

> It's currently not possible to mirror another private registry. Only the central Hub can be mirrored.

If you have any suggestions for better wording, please let me know!

Reply:
Ah, perfect! Thanks for the clarification. 🙂